project2/styles.css

```diff
@@ -0,0 +1,173 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Playfair Display', serif;
+    font-size: 17px;
+}
+
+
+.menu li {
+    display: inline-block;
+}
+
+ul {
+    text-align: left;
+    list-style: square;
+    padding-left: 60px;
+    padding-right: 40px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+.menu li+li {
+    margin-left: 110px;
+}
+
+li a {
+    background-color: rgb(255, 255, 255);
+    font-family: monospace;
+    color: orange;
+    font-size: 35px;
+    padding-left: 32px;
+
+}
+
+#fas fa-shopping-cart,
+#fas fa-utensil-spoon,
+#far fa-clock {
+    display: none;
+}
+
+#fas fa-shopping-cart:target,
+#fas fa-utensil-spoon:target,
+#far fa-clock:target {
+    display: block;
+}
+
+
+
+.fa-shopping-cart:hover {
+    color: rgb(0, 141, 47);
+}
+
+.fa-utensil-spoon:hover {
+    color: rgb(0, 141, 47);
+}
+
+.fa-clock:hover {
+    color: rgb(0, 141, 47);
+}
+
+h2 {
+    font-size: 22px;
+    text-align: left;
+    padding-left: 40px;
+    padding-top: 5px;
+}
+
+p {
+    text-align: left;
+    font-size: 20px;
+}
+
+ol {
+    padding-left: 60px;
+    padding-right: 40px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+h1 {
+    font-family: 'Hepta Slab', serif;
+    color: rgb(0, 141, 47);
+    text-align: center;
+    font-size: 40px;
+    font-style: italic;
+    font-weight: bold;
+}
+
+
+
+.fab.fa-twitter {
```

Comment:
You can simplify the styles for your social icons by
1. combining the bits that are shared
1. only using a single selector instead of the combined ones (`.fab.fa-twitter`)
like so:
```
.fab,
.fa-envelope {
  font-size: 40px;
  color: orange;
}

.fa-twitter {
  padding-left: 200px;
  padding-top: 50px;
  padding-bottom: 20px;
}

.fa-twitter:hover {
  color: rgb(55, 190, 243);
}

.fa-facebook-f:hover {
  color: rgb(0, 77, 221);
}

.fa-instagram:hover {
  color: rgb(241, 0, 181);
}

.fa-pinterest-square:hover {
  color: rgb(255, 0, 0);
}

.fa-envelope:hover {
  color: rgb(153, 0, 255);
}
```

You would also be able to eliminate some of the unique styles on the twitter icon if this was a list of social icons, because then you could use `text-align: center;` on the list 👍